config/operator/deployment/kustomization.yaml

```diff
@@ -12,5 +12,138 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: quay.io/openstack-k8s-operators/openstack-operator
-  newTag: latest
+  newName: quay.io/fpantano/openstack-operator
+  newTag: v0.0.6
```

Comment:
changes to this file should probably not be commited?

Reply:
yep, I'll revert this